airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -128,8 +128,9 @@
 - sourceDefinitionId: 9da77001-af33-4bcd-be46-6252bf9342b9
   name: Shopify
   dockerRepository: airbyte/source-shopify
-  dockerImageTag: 0.1.5
+  dockerImageTag: 0.1.6
   documentationUrl: https://docs.airbyte.io/integrations/sources/shopify
+  icon: shopify.svg
```

Comment:
can you remove the icon field for now? there is a breaking bug related to handling new icons on old airbyte versions. We need to give it some time until all consumers are migrated off the old version.